src/components/common/Legend.js

```diff
@@ -37,7 +31,7 @@ const Legend = ({ id }) => {
   const renderVerbList = () => {
     if (verbList.length !== 0) {
       return verbList
-        .reverse()
+        .sort()
```

Comment:
Can we do the sorting in the `mapStateToProps`? To avoid sorting on every render.